src/server/api/lib/twilio.js

```diff
@@ -159,14 +160,15 @@ async function sendMessage(message) {
 async function handleDeliveryReport(report) {
   const messageSid = report.MessageSid
   if (messageSid) {
+    await Log.save({ message_sid: report.MessageSid, body: report })
```

Comment:
will this work on non-postgres (i.e. non JSON-supporting) dbs?  I think we might be safer if `body: JSON.stringify(report)`